### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OatBran**
Added detailed 7th Heaven installation and answers frequently asked questions regarding FFNx and its use.
Removed old short-hand 7th Heaven install notes.
Cleaned up some markdown syntax errors and small clarifications / grammar.

### CODE REVIEW

**Comment by OatBran** [comment]
I hope I did that right, never made a PR before lol

**Comment by OatBran** [comment]
> 
> 
> This is an insane amount of info for one specific use case that goes way beyond just getting FFNx working for them. It makes the read me way to long, IMHO, This would be better in a wiki or md the read me links to for that section after a brief explanation of how to set it up .. i.e copy the driver file and shader to this path run the reg files, profit

I dont know how to set a wiki up for a repository, I dont really care if it gets merged or not, but all of this information and added clarity is needed somewhere because there are WAY too many repeat questions regarding the extremely simple process of setting it up

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ In either way, in order to use this driver you MUST have a legal copy of the gam
 
 **Supported Languages:** EN, DE, FR, SP, JP*
 
-> *: Japanese support is currently work in progress. The game starts fine but font is not rendering properly and battles do crash sometimes.
+> *: Japanese support is currently work in progress. The game starts fine but font is not rendering properly and battles do crash sometimes.*
```
Comment (julianxhokaxhiu) [inline README.md:75]: The star is a recall. Why adding it twice?

File: README.md [hunk 2]
```diff
@@ -91,6 +91,7 @@ In either way, in order to use this driver you MUST have a legal copy of the gam
 4. Extract the ZIP content next to your `ff7_*.exe` file ( for eg. for EN language `ff7_en.exe`)
 5. Replace all files when asked.
 6. Enjoy!
+   - Trying to expand your enjoyment with content mods? [Read the 7th Heaven Guide](https://github.com/julianxhokaxhiu/FFNx#FF7-7thHeaven-Steam-Conversion)
```
Comment (julianxhokaxhiu) [inline README.md:94]: No sorry, this goes beyond the scope of FFNx.

**Pull request closed without merging**